packages/dev/core/src/Materials/uniformBufferEffectCommonAccessor.ts

```diff
@@ -1,4 +1,4 @@
-import type { IColor3Like, IColor4Like, IMatrixLike, IVector3Like, IVector4Like } from "../Maths/math.like";
+import type { IMatrixLike, IVector3Like, IVector4Like, IColor3Like, IColor4Like } from "../Maths/math.like";

```

Comment:
Let s revert to previous version of the file

Reply:
Sorry for not pushing 75fef20 earlier.